core-services/image-mirroring/openshift/mapping_origin_4_11

```diff
@@ -1,3 +1,7 @@
+registry.ci.openshift.org/origin/4.11:agent-installer-api quay.io/openshift/origin-agent-installer-api:4.11 quay.io/openshift/origin-agent-installer-api:4.11.0 quay.io/openshift/origin-agent-installer-api:latest
```

Comment:
is this an api server, or just the definition of the apis (e.g. CRDs)?

Reply:
api server. The "backend" with most of the logic

Reply:
any objection to `agent-installer-api-server` then?

Reply:
No objections from me that makes it more clear.